openfood_explorer_lib.py

```diff
@@ -1,17 +1,27 @@
+import logging
+import os
 import requests
 import json
 
 from .openfood_env import EXPLORER_URL
 from .openfood_komodo_node import decoderawtransaction_wrapper
+from .logs.config import *
+
+setup_logger('explorer_libs', os.path.dirname(os.path.realpath(__file__)) + '/logs/explorer_lib_logs.log')
+logger = logging.getLogger('explorer_libs.module')
 
 def explorer_get_utxos(querywallet):
     print("Get UTXO for wallet " + querywallet)
+    logger.info("start explorer_get_utxos")
     # INSIGHT_API_KOMODO_ADDRESS_UTXO = "insight-api-komodo/addrs/{querywallet}/utxo"
     INSIGHT_API_KOMODO_ADDRESS_UTXO = "insight-api-komodo/addrs/" + querywallet + "/utxo"
     try:
-        res = requests.get(EXPLORER_URL + INSIGHT_API_KOMODO_ADDRESS_UTXO)
-        #res = requests.get(EXPLORER_URL + 'insight-api-komodo/addrs/RH5dNSsN3k4wfHZ2zbNBqtAQ9hJyVJWy4r/utxo')
+        #res = requests.get(EXPLORER_URL + INSIGHT_API_KOMODO_ADDRESS_UTXO)
+        res = requests.get(EXPLORER_URL + 'insight-api-komodo/addrs/RH5dNSsN3k4wfHZ2zbNBqtAQ9hJyVJWy4r/utxo')
```

Comment:
@ekoiav you left me a surprise!

Reply:
@imylomylo sorry. i fixed it with the latest PR

> @ekoiav you left me a surprise!